scene/gui/container.cpp

```diff
@@ -95,6 +95,7 @@ void Container::_sort_children() {
 
 void Container::fit_child_in_rect(Control *p_child, const Rect2 &p_rect) {
 
+	ERR_FAIL_COND(p_child != null);
```

Comment:
That's wrong, it's going to through an error when `p_child != null`, i.e. when the method was called correctly.

Reply:
Whoops, too used to assertions. Fixed.